Source/MediaStorage.swift

```diff
@@ -236,7 +239,52 @@ public class MediaStorage {
             completionHandler(MediaContent(data: imageData), MediaStorageError(statusCode: nil, message: nil))
         }
     }
-
+    
+    public func addMeta(mediaId mediaId: String, userMeta: Dictionary<String, String>, completionHandler: MediaStorageError -> Void) {
+        if accessToken == nil {
+            completionHandler(
+                MediaStorageError(statusCode: nil, message: "wrong usage: use the connect method to get an access token.")
+            )
+            return
+        }
+        
+        for (key, value) in userMeta {
```

Comment:
I think this method should return a `completionHandler` with or without some error message if `userMeta` is an empty `Dictionary`.